ui/src/shared/constants/index.js

```diff
@@ -445,3 +442,5 @@ export const cellSupportsAnnotations = cellType => {
   ]
   return !!supportedTypes.find(type => type === cellType)
 }
+
+export const NOTIFICATION_FADE_DURATION = 1000
```

Comment:
1. is this used anywhere?
1. and how long will the delay be for those that previously had `NOTIFICATION_DISMISS_DELAY`? `1000` sounds very short, whereas `4000` felt like a good amount of time to show a notification generally.